dom.go

```diff
@@ -161,7 +161,8 @@ func (h *HTML) restoreHTML(prev *HTML) {
 
 // Restore implements the Restorer interface.
 func (h *HTML) Restore(old ComponentOrHTML) {
-	for _, l := range h.eventListeners {
+	for _, lrange := range h.eventListeners {
+		l := lrange
```

Comment:
Why not just `l := l`? It's a well known pattern, and the idiomatic way of doing this.

> It may seem odd to write
>
> ```Go
> req := req
> ```
>
> but it's legal and idiomatic in Go to do this. You get a fresh version of the variable with the same name, deliberately shadowing the loop variable locally but unique to each goroutine.

Source: https://golang.org/doc/effective_go.html#channels

Also, `lrange` violates the [Mixed Caps](https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps) naming convention, it would be `lRange`.